.travis.yml

```diff
@@ -16,20 +16,24 @@ solution: Dlr.sln
 sudo: required
 dist: trusty
 
-dotnet: 2.0.0-preview2-006497
+dotnet: 2.0.0
 
 script:
   - dotnet restore Dlr.sln --packages ./packages
   - msbuild Build.proj /t:Build "/p:Mono=true;BuildFlavour=$CONFIGURATION"
   - pushd bin/$CONFIGURATION
   - mono ./net40/TestInternalDLR.exe
   - mono ./net45/TestInternalDLR.exe
+  - mono ./netcoreapp2.0/TestInternalDLR.exe
   - mono ./net40/Metadata.exe
   - mono ./net45/Metadata.exe
+  - mono ./netcoreapp2.0/Metadata.exe
   - mono ./../../packages/nunit.consolerunner/3.6.1/tools/nunit3-console.exe ./net40/Microsoft.Dynamic.Test.dll
   - mono ./../../packages/nunit.consolerunner/3.6.1/tools/nunit3-console.exe ./net45/Microsoft.Dynamic.Test.dll
+  - mono ./../../packages/nunit.consolerunner/3.6.1/tools/nunit3-console.exe ./netcoreapp2.0/Microsoft.Dynamic.Test.dll
```

Comment:
I'm don't think the runner will work with .NET Core. We could convert them to be console apps instead (like IronPythonTest), but I'm also fine with skipping these tests on .NET Core for now.

Reply:
Yes, seems like the NUnit team is still working on this. Maybe we should wait for them, instead of adding console projects for now.